packages/react-core/src/components/Alert/AlertActionLink.tsx

```diff
@@ -7,7 +7,7 @@ import { Button, ButtonVariant, ButtonProps } from '../Button';
 
 export interface AlertActionLinkProps extends ButtonProps {
   /** Content rendered inside the alert action link.  */
-  children?: string;
+  children?: string | React.ReactNode;
```

Comment:
Related to the above, we could update the description here just to offer a warning, "Content rendered inside the alert action link. You should not pass in interactive content such as anchor elements." I'll also open a followup in Org to update the a11y docs to mention this.

We should be able to just type this as `children?: React.ReactNode` as well since that'll include the string type, though we do have some other components that type a prop as `string | React.ReactNode` (@tlabaj I imagine we could update those instances as a followup?)

Reply:
@thatblindgeye that is correct  React.ReactNode includes string so `string` ins not needed here. We can open a up a follow up issue to clean up other interface that have the redundant definition.